src/daomop/storage.py

```diff
@@ -571,6 +567,32 @@ def cutout(self, cutout, return_file=False):
     def zeropoint(self):
         return float(self.header.get(ZEROPOINT_KEYWORD, 30.0))
 
+    @classmethod
+    def from_frame(cls, frame):
+        """
+        Takes frame number from an .ast file line and creates a FitsImage object.
```

Comment:
I think this method works regardless of where the   frame string comes from so perhaps remove the .ast here.   Might also include in this help string a short description of what a frame number is.